You might wat to check this/nderstand it
https://github.com/prisma/e2e-tests/runs/4775213666?check_suite_focus=true#step:6:131
```
WARNING in ./node_modules/@prisma/client/runtime/index.js 14993:18-45
Module not found: Error: Can't resolve 'encoding' in '/home/runner/work/e2e-tests/e2e-tests/bundlers/webpack/node_modules/@prisma/client/runtime'
 @ ./node_modules/.prisma/client/index.js 17:4-43 134:29-68 59:21-54
 @ ./node_modules/@prisma/client/index.js 1:15-46
 @ ./src/index.js 1:0-45 3:19-31

WARNING in configuration
The 'mode' option has not been set, webpack will fallback to 'production' for this value.
Set 'mode' option to 'development' or 'production' to enable defaults for each environment.
You can also set it to 'none' to disable any default behavior. Learn more: webpack.js.org/configuration/mode
```

--- a/bundlers/webpack/webpack.config.js
+++ b/bundlers/webpack/webpack.config.js
@@ -2,6 +2,7 @@ module.exports = {
   target: 'node',
   externals: {
     "_http_common": "commonjs2 _http_common",
+    "encoding": "commonjs2 encoding",
   }
 }
 

